src/modules/wishlist/routes/wishlist.route.tsx

```diff
@@ -0,0 +1,49 @@
+import Box from '@mui/material/Box';
+import { useEffect, useState } from 'react';
+import { Wishlist } from '../components/wishlist';
+import Typography from '@mui/material/Typography';
+
+const styles = {
+  shadowBox: (showShadow: boolean) => ({
+    height: '2px',
+    position: 'fixed',
+    top: `var(--header-height)`,
+    left: 0,
+    right: 0,
+    zIndex: 10,
+    backgroundColor: '#fff',
+    borderBottom: '1px solid #ddd',
+    boxShadow: showShadow ? '0px 4px 8px rgba(0, 0, 0, 0.2)' : 'none',
+    transition: 'box-shadow 0.3s ease',
+  }),
+
+  header: {
+    marginTop: '16px',
+    marginBottom: '16px',
+    fontWeight: 'bold',
+    fontSize: '28px',
+    color: '#333',
+  },
+};
+
+export function WishlistRoute() {
+  const [showShadow, setShowShadow] = useState(false);
+
+  useEffect(() => {
```

Comment:
Why it's needed?

Reply:
I assume the purpose is to add an event listener that will display this shadow on top of the container when the user scrolls down.

![image](https://github.com/user-attachments/assets/6ce9b848-2be3-4bdb-af13-50c6071a769e)

Reply:
Yeah, but now we've decided to keep the styling simpler